External/FEXCore/Source/Utils/MemberFunctionToPointer.h

```diff
@@ -17,12 +17,13 @@ class MemberFunctionToPointerCast final {
     MemberFunctionToPointerCast(PointerToMemberType Function) {
       memcpy(&PMF, &Function, sizeof(PMF));
 
-#ifdef _M_X86_64
+#if defined(_M_X86_64) || defined(_M_RISCV_64)
+      // XXX: Is RISCV iatnium?
```

Comment:
```suggestion
      // XXX: Is RISCV Itanium?
```

Reply:
Also, what kind of question is that??   No, Itanium is IA64, an Intel proprietary (and failed) ISA. RISC-V is an utterly different open source ISA.

Reply:
It's not a question about the ISA being Itanium. It's a question if RISC-V follows the Itanium C++ ABI style of member function pointers.
Itanium spec around this is here: https://itanium-cxx-abi.github.io/cxx-abi/abi.html#member-function-pointers

The question is raised since Member Function pointer spec in most architectures follow exactly what Itanium does here, but AArch64 doesn't. Which is why we need to find the RISC-V spec on this. The question being for future me.

Thanks for the code review.

Reply:
Oh I see. Maybe the comment could have been clearer :-)

I don't know of anywhere that it is explicitly stated, but it appears to be implicit that RISC-V doesn't intend to break any new ground and cause incompatibility with other software here. Certainly function pointers are just pointers, the low bit can't be set, VTBL entries can't have an odd size and so forth.

RISC-V founder Andrew Waterman references the Itanium ABI with respect to C++ classes with nontrivial copy constructors here: https://github.com/riscv-non-isa/riscv-elf-psabi-doc/issues/7

RISC-V is explicitly documented as using the same C++ name-mangling as Itanium: https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-elf.adoc#c-name-mangling

You could open an issue asking for confirmation in that repo, but the intent seems clear.